### PR TITLE
Add Convert String to Date Task

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: src/main/java/core/utils/Daily.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+/**
+ * Gets us predefined values of different time units.
+ */
+public enum Daily {
+    today("day", 0),
+    yesterday("day", -1),
+    tomorrow("day", 1),
+    ago("ago", -1),
+    from("from", 1);
+    String daily;
```
Comment (danielaSantaCruz) [inline src/main/java/core/utils/Daily.java]: these variables could be private

File: src/main/java/core/utils/DateConverter.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+     *
+     * @param phrase is a string that we want to transform.
+     * @return a date entity.
+     */
+    public static Date transform(String phrase) {
+        if (isCorrect(phrase)) {
+            try {
+                return createDate(phrase);
+            } catch (ParseException e) {
+                e.printStackTrace();
```
Comment (danielaSantaCruz) [inline src/main/java/core/utils/DateConverter.java]: You should handle the exception better, not just printing it.

File: src/main/java/core/utils/DateConverter.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    }
+
+    /**
+     * Calculates a date given time units amount and type.
+     *
+     * @param election is an integer that represents type.
+     * @param value is an integer that represents amount.
+     * @return a date entity.
+     */
+    public static Date calculate(int election, int value) {
```
Comment (danielaSantaCruz) [inline src/main/java/core/utils/DateConverter.java]: Maybe some methods could be private or protected instead od public

File: src/main/java/core/utils/DateConverter.java [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        Date date = new Date();
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(date);
+        calendar.add(election, value);
+        return date = calendar.getTime();
```
Comment (danielaSantaCruz) [inline src/main/java/core/utils/DateConverter.java]: You could directly return the value.

File: src/main/java/core/utils/Daily.java (hunk 1 above)
Comment (JRojas999) [reply src/main/java/core/utils/Daily.java]: You were right. I changed it.

File: src/main/java/core/utils/DateConverter.java (hunk 3 above)
Comment (JRojas999) [reply src/main/java/core/utils/DateConverter.java]: You were right. I changed it.

File: src/main/java/core/utils/DateConverter.java (hunk 4 above)
Comment (JRojas999) [reply src/main/java/core/utils/DateConverter.java]: Yeah it is redundant. I alreasy changed it.

**Review by cristianfchq — APPROVED** [review]

**Comment by cristianfchq** [comment]
good job

**Pull request merged**